R/api-data-checker.r

```diff
@@ -0,0 +1,97 @@
+# This is a standalone function not run by the screener itself.
+# I'm using it to quickly create a list of column names and filter items from
+# a set of data files and sort them into a data-dictionary structure with
+# flags as to whether they're currently in the data dictionary or not.
+
+api_data_checker <- function(files) {
+  entries <- data.frame(
+    col_name = NA,
+    col_type = NA,
+    filter_item = NA,
+    col_name_parent = NA,
+    filter_item_parent = NA
+  )
+  for (file in files) {
+    data <- vroom::vroom(file)
+    meta <- vroom::vroom(gsub(".csv", ".meta.csv", file))
```

Comment:
Might be nice to reduce the console outputs?
```suggestion
    data <- vroom::vroom(file, show_col_types = FALSE)
    meta <- vroom::vroom(gsub(".csv", ".meta.csv", file), show_col_types = FALSE)
```